<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseMonitoringDlg</class>
 <widget class="QDialog" name="PauseMonitoringDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>262</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pause monitoring/alerting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qHostMonitor.qrc">
    <normaloff>:/img/hostMonitor.png</normaloff>:/img/hostMonitor.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="chkMonitoringPause">
            <property name="text">
             <string>Pause monitoring till</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateMonitoringPause">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeMonitoringPause">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>hh:mm:ss</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblMonitoringPauseGmt">
            <property name="text">
             <string>Must be GMT</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkMonitoringStopped">
            <property name="text">
             <string>Start action profile every time monitoring is stopped/paused</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="cmbMonitoringStopped">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Do nothing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Message, Sound</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reboot</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reports</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Repair server</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>change test interval / 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>day-nights alerts</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnMonitoringStopped">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../qHostMonitor.qrc">
                <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkMonitoringStarted">
            <property name="text">
             <string>Start action profile every time monitoring is started/resumed</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="cmbMonitoringStarted">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Do nothing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Message, Sound</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reboot</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reports</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Repair server</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>change test interval / 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>day-nights alerts</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnMonitoringStarted">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../qHostMonitor.qrc">
                <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="chkAlertingPause">
            <property name="text">
             <string>Pause alerting till    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateAlertingPause">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeAlertingPause">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>hh:mm:ss </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblAlertingPause">
            <property name="text">
             <string>Must be GMT</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkAlertingStopped">
            <property name="text">
             <string>Start action profile every time alerting is stopped/paused     </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="cmbAlertingStopped">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Do nothing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Message, Sound</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reboot</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reports</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Repair server</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>change test interval / 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>day-nights alerts</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnAlertingStopped">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../qHostMonitor.qrc">
                <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkAlertingStarted">
            <property name="text">
             <string>Start action profile every time alerting is started/resumed</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="cmbAlertingStarted">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Do nothing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Message, Sound</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reboot</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>reports</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Repair server</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>change test interval / 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>day-nights alerts</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnAlertingStarted">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../qHostMonitor.qrc">
                <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QDialogButtonBox" name="btnBoxPauseMonitoring">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>140</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBoxPauseMonitoring</sender>
   <signal>rejected()</signal>
   <receiver>PauseMonitoringDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonitoringPause</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateMonitoringPause</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonitoringPause</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeMonitoringPause</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonitoringStopped</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbMonitoringStopped</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonitoringStopped</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnMonitoringStopped</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonitoringStarted</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbMonitoringStarted</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMonitoringStarted</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnMonitoringStarted</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAlertingPause</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateAlertingPause</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAlertingPause</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeAlertingPause</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAlertingStopped</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAlertingStopped</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAlertingStarted</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbAlertingStarted</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAlertingStopped</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnAlertingStopped</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>579</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAlertingStarted</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnAlertingStarted</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
